### ISSUE TITLE
Warning on install due to rubyzip2 being deprecated

### ISSUE THREAD

**Issue by rkettelerij** [cloudfoundry-attic/vmc#41]
Installing vmc (on a fresh OS X install) results in the following warning. This due to rubyzip2 being deprecated.

```
rickette-macbook:travel rickette$ sudo gem install vmc
Password:

DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED!
DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED!

 \ The rubyzip2 gem is deprecated. Instead, please use:

  * rubyzip       https://github.com/aussiegeek/rubyzip
 \ * archive-zip   https://github.com/javanthropus/archive-zip

DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED!
DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED! DEPRECATED!
```

**Comment by leto** [comment]
Looks like we fixed this in #38, perhaps you are using an out-of-date repo?

**Comment by rkettelerij** [comment]
Closing this issue, since it's fixed in #38.
